src/main/cpp/src/utils/logger.cc

```diff
@@ -43,7 +43,7 @@ Logger logger;
 Logger::Logger() {
   m_logger = spdlog::get(LOGGER_NAME);
   if (m_logger == nullptr) {
-    m_logger = spdlog::stdout_color_mt(LOGGER_NAME);
+    m_logger = spdlog::stderr_color_mt(LOGGER_NAME);
```

Comment:
This was the error that wasn't being caught in previous PRs.

Reply:
My bad 👎 . Thanks for fixing.